sdk/dapp-kit/src/components/connect-modal/ConnectModal.tsx

```diff
@@ -67,8 +93,8 @@ export function ConnectModal({ trigger }: ConnectModalProps) {
 	}
 
 	return (
-		<Dialog.Root open={isConnectModalOpen} onOpenChange={onOpenChange}>
-			<Dialog.Trigger className={styles.triggerButton}>{trigger}</Dialog.Trigger>
```

Comment:
I'm removing these styles for this PR just because I'm adding a `Button` primitive in the next PR for theming that'll handle styling the button for the `ConnectButton` component